Edited a Section to be less Wordy (#36)
comp129/comp55: docs/labs/12-Hash.md

## But...what if there are multiple meanings for the same word?

If you want to store multiple meanings,
you'll have to select an object or class that will allow you to do so.
For example,
you can have a word with multiple, sometimes even contradictory meanings.
Let's start up another ```HashMap```,
this time for contronyms which are words that often have opposite meanings.
Think about the word ***dust***.
Usually, it means to take off something, like to *dust* the furniture.
However,
the word *dust* can also mean to add something instead of taking it away!

> An example is that you may want to *dust* a pan with flour.

Reviewer: ```suggestion
For example, you may want to *dust* a pan with flour.
```
I'm not sure that this made it less words here

In this case, the word **dust** has two meanings.
Let's say we want to create a map that has both.
How would this be done?
We could create some long string,

> but another approach we can do is simply create a different linkage,

Reviewer: ```suggestion
but we also merely create a different linkage,
```
